testdata/scripts/syntax.txt

```diff
@@ -1,19 +1,20 @@
 env GOPRIVATE='test/main,rsc.io/*'
 
-garble build
+garble build -tags directives
```

Comment:
why do you need a build tag?

Reply:
or rather, if you need it, document why

Reply:
This is a test for correct processing // +build. I added it because the processing mechanism has changed a lot. It's a bit outside the scope of the commit theme but I think it's worth keeping this test.